src/NServiceBus.Core/Saga/SagaPersistenceOptions.cs

```diff
@@ -0,0 +1,35 @@
+namespace NServiceBus.Saga
+{
+    using NServiceBus.Extensibility;
+
+    /// <summary>
+    /// Contains details about the to be persisted saga
+    /// </summary>
+    public class SagaPersistenceOptions
+    {
+        /// <summary>
+        /// Creates a new instance of the SagaPersistenceOptions class.
+        /// </summary>
+        /// <param name="sagaMetadata">The saga metadata representing the to be persisted saga.</param>
+        /// <param name="context">The context.</param>
+        public SagaPersistenceOptions(SagaMetadata sagaMetadata, ContextBag context = null)
```

Comment:
Currently the ctor is also used for testing purposes. This allows to shortcut it and do less typing.